Bottom: 32bcea264d817bc347b447007bd2efffe6153f30
Top:    29ad88f5f8a8118889256cc8f2e07db96b0c4bb2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-17 03:05:54 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index ce4724d..fc1f37a 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -61,9 +61,9 @@ hidgl_add_triangle (triangle_buffer *buffer,
                     GLfloat x2, GLfloat y2,
                     GLfloat x3, GLfloat y3)
 {
-  hidgl_add_triangle_3D (x1, y1, global_depth,
-                         x2, y2, gloabl_depth,
-                         x3, y3, global_depth);
+  hidgl_add_triangle_3D (buffer, x1, y1, global_depth,
+                                 x2, y2, gloabl_depth,
+                                 x3, y3, global_depth);
 }
 
 // void draw_grid ()
